<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddCondition</class>
 <widget class="QDialog" name="DialogAddCondition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="4" column="2">
    <widget class="QPushButton" name="BtnDeleteCondition">
     <property name="text">
      <string>删 除</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="BtnModifyCondition">
     <property name="text">
      <string>修 改</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="BtnAddCondition">
     <property name="text">
      <string>添加条件</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="BtnOK">
     <property name="text">
      <string>确 定</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="BtnCancel">
     <property name="text">
      <string>取 消</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>条件(满足任意一个即可)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QRadioButton" name="RbModeType">
     <property name="text">
      <string>候选集包含该器件失效模式</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QRadioButton" name="RbBelongLink">
     <property name="text">
      <string>存在候选集属于该器件所在单链回路</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QRadioButton" name="RbIncludeUnit">
     <property name="text">
      <string>候选集包含该器件</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
